dandischema/models.py

```diff
@@ -988,10 +989,14 @@ class PublishedAsset(Asset, Publishable):
     @validator("digest")
     def digest_bothhashes(cls, values):
         try:
-            if len(values[DigestType.dandi_etag] + values[DigestType.sha2_256]) != 96:
-                raise ValueError
-        except (KeyError, ValueError):
-            raise ValueError("Digest must have both valid dandi-etag and sha2-256.")
+            digest = values[DigestType.dandi_etag]
+            if "-" not in digest or len(digest.split("-")[0]) != 32:
```

Comment:
I think 
```suggestion
            if len(digest.split("-")[0]) != 32:
```
would be sufficient but ok either way